processing/src/main/java/io/druid/query/dimension/RegexFilteredDimensionSpec.java

```diff
@@ -65,10 +65,6 @@ public DimensionSelector decorate(final DimensionSelector selector)
       return selector;
     }
 
-    if (selector == null) {
```

Comment:
You can be more clear as in : Code is duplicated immediately prior to here.

The duplicate code is hidden in the fold